supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/PipelinesApiAsyncBackendJobExecutionActor.scala

```diff
@@ -824,6 +825,16 @@ class PipelinesApiAsyncBackendJobExecutionActor(override val standardParams: Sta
         throw new RuntimeException(s"handleExecutionSuccess not called with RunStatus.Success. Instead got $unknown")
     }
 
+  override def getStartAndEndTimes(runStatus: StandardAsyncRunState): Option[(OffsetDateTime, OffsetDateTime)] =
+    runStatus match {
+      case terminalRunStatus: TerminalRunStatus if terminalRunStatus.eventList.nonEmpty =>
+        val offsetDateTimes = terminalRunStatus.eventList.map(_.offsetDateTime)
+        Some((offsetDateTimes.min, offsetDateTimes.max))
```

Comment:
My memory of this was incorrect when we talked, we DO want to look at the latest event for end time, but for start time we want to look for the event with description `LIKE r'Worker \\"google-pipelines-worker-% assigned in %'`. See query here: https://github.com/broadinstitute/dsp-batch-metrics/blob/main/templates/BT-512-cpu-and-memory-hours.sql#L95 That event represents the VM actually picking up the task and starting to run it.